packages/foundry.core/src/_components.ts

```diff
@@ -233,8 +233,7 @@ export type TimeUnit =
   | "DAYS"
   | "WEEKS"
   | "MONTHS"
-  | "YEARS"
-  | "QUARTERS";
+  | "YEARS";
```

Comment:
Could you track as a FLUP to see if we need to remove support for `QUARTERS` in our internal time unit types?